foundry/applications/formApplications/userManagement.d.ts

```diff
@@ -1,3 +1,70 @@
-declare class UserManagement extends FormApplication {
-  _updateObject(event?: Event, formData?: object): Promise<any>; // TODO: Add correct return type
+/**
+ * The User Management setup application
+ */
+declare class UserManagement extends FormApplication<UserManagement.Data, Users> {
+  /**
+   * @override
+   * @defaultValue
+   * ```typescript
+   * mergeObject(super.defaultOptions, {
+   *   id: "manage-players",
+   *   classes: ["dark"],
+   *   template: "templates/setup/user-management.html",
+   *   popOut: false,
+   *   closeOnSubmit: false,
+   *   scrollY: ["#player-list"]
+   * });
+   * ```
+   */
+  static get defaultOptions(): FormApplication.Options;
+
+  /** @override */
+  protected _render(force?: boolean, options?: Application.RenderOptions): Promise<void>;
```

Comment:
```ts
  protected _render(...args: Parameters<FormApplication['render']>): Promise<void>;
```

Reply:
Same as earlier, this should be done consistently and in a separate PR.